prime-angular-frontend/src/app/shared/components/site/contact-profile-form/contact-profile-form.component.html

```diff
@@ -15,9 +15,43 @@
       </ng-container>
     </app-page-subheader2>
 
-    <ng-content select="app-same-as"></ng-content>
-
-    <div class="row">
+    <ul class="list-unstyled mb-4 ml-3">
```

Comment:
Was this contact card supposed to be implemented in this ticket?  They are still making decisions around contacts, and site registration and health authority site registrations won't be the same.

Reply:
yes, confirmed with Pilar. She did not add mockup screen to the ticket though.. Will add the mockup to ticket too.

![image](https://user-images.githubusercontent.com/58282608/116731066-18be8c80-a99e-11eb-9c8a-c5b7f1ffca67.png)

Reply:
@johnnyl6 please don't make changes to the tickets acceptance criteria mid sprint even with Pilar.  This was a 1 point ticket, and if I had known I would have let you know that we weren't planning on maintaining this as a shared component between registrations

Reply:
We'll break this out into just HA in the next ticket